src/Common/tests/hash_table.cpp

```diff
@@ -1,5 +1,5 @@
-#include <iostream>
 #include <iomanip>
```

Comment:
BTW, this test is also not unitized (as it is very old), it will be better to convert it to unit test.

Reply:
Updated.